### DISCUSSION TITLE
Flow Launcher

### DISCUSSION THREAD

**Discussion by Fnige** [catppuccin/catppuccin#1902] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

https://www.flowlauncher.com/docs/#/how-to-create-a-theme

### Has the requested port been themed already?

Multiple times: https://github.com/Flow-Launcher/Flow.Launcher/discussions/1438

### Any additional comments?

Would be nice to have multiple versions of each theme (I.e Matte, Blurred etc)

**Comment by syndrizzle** [comment]
Hey, just wanted to say i've worked on it and the port is ready to be reviewed at https://github.com/Syndrizzle/flow-launcher
